web/src/app/auth/create-account/page.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
```

Comment:
Can we gate this so users don't visit this? If not, it doesn't really matter :P

Reply:
Added comment!